Automattic/wp-calypso
# Add/upsell titan

#### Changes proposed in this Pull Request

* Start showing the Titan "Get Email" banner in the primary cards area on the Home page.

- [x] Add a `TitanBanner.js` task to the primary cards

#### Testing instructions


1. Add the talk-to/wp-calypso repo as a remote - git remote add titan https://github.com/talk-to/wp-calypso.git
2. Fetch the add/upsell-titan branch from the Titan repo: git fetch titan add/upsell-titan
3. Add the following line to client/my-sites/customer-home/main.jsx on L99:
```javascript
<Primary cards={ [ 'home-task-upsell-titan' ] } />
```
4. Click on "My Home" on the sidebar. You should see the Titan Upsell Banner on the Customer Home page
#### Caveats
The task `home-task-upsell-titan` will be obtained from the server.

#### Screenshot
Titan Upsell Banner on Customer Home Page:
![image](https://user-images.githubusercontent.com/1598017/118619977-b3163100-b7e2-11eb-898b-578419fb90a0.png)


#### Resources
* Design resources: https://www.figma.com/proto/h9Vnr8v7bYB7fvIJ9FvGYs/Customer-Home-Checklist-(Copy)?node-id=763%3A1&viewport=282%2C308%2C0.06280452013015747&scaling=scale-down&page-id=759%3A16689
* pcDL0W-ad-p2
* pcDL0W-ar-p2


cc: @daledupreez
